src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -530,32 +531,16 @@ def test_WINNF_FT_S_PCR_2(self, config_filename):
   def generate_PCR_3_default_config(self, filename):
     """Generate the WinnForum configuration for PCR 3."""
     # File path where SAS UUT claimed ppa boundary generated in PCR.1 test
+    pcr_1_test_config_file_name = 'default.config'
     pcr_1_test_config_file_path = os.path.join('testcases', 'configs',
```

Comment:
Can this be removed?

Reply:
Extra   variable representing the 'default.config' has been removed.

Reply:
I meant: are you using `pcr_1_test_config_file_path` anywhere? I think you can delete *that* variable.

Reply:
I think pcr_1_test_config_file_path is being used in generate_PCR_3_default_config:
    pcr_1_test_config_file_path = os.path.join('testcases', 'configs',
                                               'test_WINNF_FT_S_PCR_1',
                                               'default.config')

    config = {
        'configPCR_1': pcr_1_test_config_file_path,
        'userClaimedPpaContour': user_claimed_ppa_contour_geometry
    }

and in test_WINNF_FT_S_PCR_3:
    pcr_1_test_config = loadConfig(config['configPCR_1'])

Reply:
Oops, good point! Thank you :) Merge when ready!